### ISSUE TITLE
‘static func stories<C: Collection>(by identifiers: C)’ always return error.

### ISSUE THREAD

**Issue by valentizx** [sbertix/Swiftagram#59]
- [x] I've searched past issues and I couldn't find reference to this. 

Hi Sbertix,

I  try to get **reelMedia** list by:
```
static func stories<C: Collection>(by identifiers: C) -> Endpoint.ResponseDisposable where C.Element == String
```
But no matter I if the parameter I pass in is `archived media's id` or `user's pk`:
```
Endpoint.Feed.stories(by:["archiveDay:18038182546262472","archiveDay:17918118640295965"])
                .unlocking(with: secret)
                .task(
                    by: .instagram,
                    onComplete: { result in
                        switch result {
                        case .success(let response):
                            print(response)
                        case .failure(let error):
                            print(error)
                        }
                }).resume()

or

Endpoint.Feed.stories(by:[my.pk])
                .unlocking(with: secret)
                .task(
                    by: .instagram,
                    onComplete: { result in
                        switch result {
                        case .success(let response):
                            print(response)
                        case .failure(let error):
                            print(error)
                        }
                }).resume()
```
It always return an error like this:
```
["status": Response("fail"), "message": Response("Invalid reel id list")]
```

In [SwiftyInsta](https://github.com/TheM4hd1/SwiftyInsta) 
Sometimes I get the correct result, sometimes I get empty data.

Sorry 4 my poor English~

**Comment by sbertix** [comment]
Hey @valentizx 😊\
I'm gonna look into this ASAP and update you.

**Comment by sbertix** [comment]
The changes in #60 made it work for me.\
I'm merging it right now and bumping to `3.1.1`.\
Let me know if it fixes for you 💪

**Comment by sbertix** [comment]
_If you found this useful, please consider starring :star: this repo, or even [sponsoring it](https://paypal.me/sbertix) :dollar:.<br />Even just $5 can help me maintain this._<br />_Thank you for using **Swiftagram**._

**Comment by sbertix** [comment]
I've found out that the error can still occur on `4.0.*`.\
I've pushed a fix that will be released with `4.1.0`.\
Sorry.
